source/standards/cdio-pillars.html.md.erb

```diff
@@ -0,0 +1,100 @@
+---
+title: CDIO Pillars
+last_reviewed_on: 2020-08-26
+review_in: 3 months
+---
+
+# Chief Digital and Information Office Teams
+
+The Chief Digital and Information Office (CDIO) pillars each represent a number of teams that work in a given area, some of these teams came from GDS or the Cabinet Office and some were newly created to meet the needs of the CDIO.
+
+We work with GDS, CDIO and Cabinet Office technology teams to:
+
+* provide a common set of tools to support common problems
+* integrate and configure those tools
+* build and maintain a secure platform making it easy for teams to build services
+* provide Security Operations, Engineering and Architecture support
+
+## Change and Deliver
+
+Change and Deliver contains the Reliability Engineering, Digital Marketplace and GovWifi Teams. It
+provides shared tools and services. This includes support for:
+
+* Amazon Web Services (AWS) accounts
+* [Logit][], for storing and querying infrastructure and application logs
+* [Prometheus][], for running an operational metrics service
+* Our Continuous integration service
+* Other GDS services and teams
```

Comment:
Capitalisation of the bullets needs to be consistent - first letter is lower case unless it's a proper noun (also in the other bullet lists)